doc/integrator/checker.rst

```diff
@@ -3,41 +3,40 @@
 Automated check
 ===============
 
-c2cgeoportal applications include web services for testing
+c2cgeoportal applications include functionality for testing
 and assessing that the application is correctly functioning,
 ie. that its web services respond as expected.
 
-For that we have two services: a *checker* and a *check_collector*.
+For that we have parts: a *checker* and a *check_collector*.
 
-Those services (especially the collector) are meant to be used by a
+Those (especially the collector) are meant to be used by a
 monitoring system like Nagios to check that the application is alive.
 
-The return code are:
+They integrate into c2cwsgiutils's health check service. The return code are:
 
 * ``200``-``299`` => OK
-* ``400``-``499`` => Warning
```

Comment:
Why removing warning?

Reply:
There is no such thing as warning in health_check

Reply:
=> 500 or 200